src/test/it/essentials.yml

```diff
@@ -0,0 +1,19 @@
+---
+ath:
+  useLocalSnapshots: false
+  athRevision: "dad333092159cb368efc2f9869572f0a05d255ac"
+  tests:
+    - "plugins.SshSlavesPluginTest"
+    - "core.SlaveTest"
+  categories:
+    - "org.jenkinsci.test.acceptance.junit.SmokeTest"
```

Comment:
Tests in this category are in addition to the specified `tests` above, I presume?

Reply:
yes